sdk/metric/benchmark_test.go

```diff
@@ -42,7 +42,9 @@ var viewBenchmarks = []struct {
 		"AttrFilterView",
 		[]View{NewView(
 			Instrument{Name: "*"},
-			Stream{AllowAttributeKeys: []attribute.Key{"K"}},
+			Stream{AttributeFilter: func(kv attribute.KeyValue) bool {
+				return kv.Key == attribute.Key("K")
+			}},
```

Comment:
While this is probably more optimized, how does it compare with using a `NewAllowKeysFilter("K")`?

If they are in the same ballpark, I recommend using the functions we created for this.

Reply:
Sure, sounds good. I had just reverted, but using the new func sounds good to me.